projects/github-actions/repo-gardening/src/tasks/triage-new-issues/index.js

```diff
@@ -93,6 +111,46 @@ function findPriority( body ) {
 	return null;
 }
 
+/**
+ * Build an object containing the slack message and its formatting to send to Slack.
+ *
+ * @param {WebhookPayloadIssue} payload - Issue event payload.
+ * @param {string}              channel - Slack channel ID.
+ * @param {string}              message - Basic message (without the formatting).
+ * @returns {object} Object containing the slack message and its formatting.
+ */
+function formatSlackMessage( payload, channel, message ) {
```

Comment:
I may want to extract this function into a helper that can be shared by the different tasks, since I have a few variations of it already.